### DISCUSSION TITLE
Easily Moddable UI for flag variables, making it similar like how fuel and water tank's UI look

### DISCUSSION THREAD

**Discussion by Ongjming** [SmartlyDressedGames/Unturned-3.x-Community#2672] [Ideas]
I was just thinking about this randomly and thought that it might be good to be able to easily mod some visibility for things that are usually not visible (Like quest flag variables for variety of usage/meaning)

The current water tank UI has some short description with a colored bar below and the small icon to the left of that bar.
The Fuel tank is similar except for the color difference and icon difference.

If there is an easy way to mod in this as an indicator (Maybe a quest score board thingy? Quest checklist for personal reference of quests' completion status? Custom resource that can go up and down, etc.) there would be lots of posibilities :D

What I had in mind on the player modders' side:
Find an icon for the left of the bar, chose a color for the bar, add a description.
(Maybe an image upload for the icon, .dat file for hex color + description information.)

Of course I have no idea how the game works in general for this to be as easy as I have thought might be possible..... pardon me as I am truly that ignorant on that end.
(This idea came about after seeing the water and fuel UI bar for quite a while)

**Comment by SDGNelson** [comment]
Shown in the tracked quest and quest menus? I agree that would look nice.
